### ISSUE TITLE
Issue with 02958343 training data

### ISSUE THREAD

**Issue by BenJourdan** [eldar/differentiable-point-clouds#7]
I tried training the model using the 02958343 data and found at least one training step had some issues:

On step 655 some values were set to zero which led to a divide by zero error in function  `ypr_from_campos` in `util/euler.py` , making the loss nan and ruining the training.

I found a quick fix was just to check for this in the function and return 0,0,0 if this happened ( 0 yaw, pitch, roll).

![image](https://user-images.githubusercontent.com/13700535/71412736-e92f9f80-2646-11ea-8b70-efb72ba36f95.png)
